sql/hive/src/test/resources/golden/timestamp cast #3-0-89ef5480d0dd001cec4642de58f8632f

```diff
@@ -0,0 +1 @@
+1.0
```

Comment:
which test produces this golden file?

Reply:
This test case "timestamp cast #3" 
before it is used for testing casting double to timestamp.